### ISSUE TITLE
[Minor] Biome colours are weird

### ISSUE THREAD

**Issue by Hyxaru** [TheCodex6824/ThaumicAugmentation#271]
There seems to be stripes going across the north/west - south/east diagonals. More so, it seems that on the negative coordinates/side the biome colors are much darker
![2022-01-09_14 35 44](https://user-images.githubusercontent.com/11833817/148684562-6f3264ed-c638-4755-b451-623953e64ac0.png)
.

**Comment by TheCodex6824** [comment]
Yeah, that will probably be something I get to when/if I ever get around to the Emptiness worldgen revamp
